hrjobcontract/js/src/job-contract/controllers/modal/modal-contract.js

```diff
@@ -53,6 +54,9 @@ define([
             angular.copy(entity,$scope.entity);
             angular.copy(files,$scope.files);
 
+            $scope.entity.details.period_start_date = convertToDateObject($scope.entity.details.period_start_date);
```

Comment:
Yeah, I see now. I somehow missed it (maybe hadn't enough coffee...) :(
Approved :)